docs/walkthrough.md

```diff
@@ -138,17 +138,31 @@ chmod +x ./kubectl
 We'll assume hereafter that all `kubectl` commands are using this
 newly-installed executable.
 
-To configure `kubectl` to communicate with the service catalog APU server:
+## Step 5 - Configuring `kubectl` to Talk to the API Server
+
+To configure `kubectl` to communicate with the service catalog API server, we'll have to
+get the IP address that points to the `Service` that sits in front of the API server pod(s).
+If you installed the catalog with one of the `helm install` commands above, then this service 
+will be called `catalog-catalog-apiserver`, and be in the `catalog` namespace. 
+
+### Notes on Getting the IP Address
+
+How you get this IP address is highly dependent on your Kubernetes installation method. Regardless
+of how you do it, do not use the Cluster IP of the `Service`. The `Service` is created as a
+`NodePort` in this walkthrough, so you'll likely need to use the IP address of the node or one of
+the nodes in your cluster.
+
+### Setting up a New `kubectl` Context
+
+When you determine the IP address of this service, set its value into the `SVC_CAT_API_SERVER_IP`
```

Comment:
I still think that adding an instruction for `minikube` would be really useful.
i.e. as @MHBauer suggested, `export SVC_CAT_API_SERVER_IP=$(minikube ip)` (or `minikube service ...`)
with a note that other installations would have different ways of determining IP. IMO that would make a walkthrough much more straightforward for tire kicking.